src/shapes/itext.class.js

```diff
@@ -255,7 +255,6 @@
      */
     _render: function(ctx) {
       this.callSuper('_render', ctx);
-      this.ctx = ctx;
     },
```

Comment:
_render does nothing except calling super, can be removed I guess

Reply:
yeah....